src/app/application/services/i18n/I18n.ts

```diff
@@ -44,7 +49,10 @@ export default <T extends I18nRecord>(strs: T, env: Env['var']) => {
         if (get(strs, key).length === 0) {
           return ''
         }
-        rest[1] = { KUMA_DOCS_URL: env('KUMA_DOCS_URL'), KUMA_UTM_QUERY_PARAMS: env('KUMA_UTM_QUERY_PARAMS'), ...rest[1] }
+        rest[1] = {
+          ...globals,
+          ...rest[1],
+        }
```

Comment:
These 'globals; never change so I just hoisted them to only retrieve the once rather than everytime we call `t`